sources/dicom/large_image_source_dicom/assetstore/dicomweb_assetstore_adapter.py

```diff
@@ -512,6 +493,63 @@ def importData(self, parent, parentType, params, progress, user, **kwargs):
 
         return items
 
+    def importData(self, parent, parentType, params, progress, user, **kwargs):
+        """
+        Import DICOMweb WSI instances from a DICOMweb server.
+
+        :param parent: The parent object to import into.
+        :param parentType: The model type of the parent object.
+        :type parentType: str
+        :param params: Additional parameters required for the import process.
+            This dictionary may include the following keys:
+
+            :limit: (optional) limit the number of studies imported.
+            :search_filters: (optional) a dictionary of additional search
+                filters to use with dicomweb_client's `search_for_series()`
+                function.
```

Comment:
Actually, this looks like it was changed a little bit. It's now `filters` instead (see below), and it is required to be a JSON string, not a dict.

The REST API will be passing these params as strings (a string for the int, and JSON for the dict). I guess if someone wanted to call this function with a dict instead, they would need to call the `_importData()` variant instead?

Reply:
I've updated this to allow for the passage of `filters` as a dictionary. The parallel concern you had with limit should be handled already